docs/backups-restore.md

```diff
@@ -49,8 +49,7 @@ spec:
   pgCluster: cluster1
   repoName: repo1
   options:
-  - --type=time
-  - --target="2022-11-30 15:12:11+03"
+  - --type=immediate
```

Comment:
I don't think we should change this example. And specifying only `--type=immediate` without `--set` most likely won't work.

Reply:
done: reverted example to the original variant